pkg/networkpath/traceroute/runner.go

```diff
@@ -8,15 +8,23 @@ package traceroute
 import (
```

Comment:
Overall testing feedback, I think we should likely start adding test coverage to network path agent code.